tests/tests_add_rm.yml

```
@@ -0,0 +1,161 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Test creating, modifying, and removing kernels
+  hosts: all
+  gather_facts: false
+  tags:
+    - tests::reboot
+  vars:
+    bootloader_reboot_ok: true
+  tasks:
+    - name: Get bootloader_facts
+      vars:
+        bootloader_gather_facts: true
+      include_role:
+        name: linux-system-roles.bootloader
+
+    # Images in CI might have a grub timeout set to a different other than the
+    # default 5 value.
+    # In this case, the above invocation require handlers to be flushed.
+    - name: Flush handlers
+      meta: flush_handlers
+
+    - name: Set fact with the default kernel to use for clones
+      set_fact:
+        test_kernel: "{{ bootloader_facts | rejectattr('initrd', 'undefined')
+          | selectattr('default', 'equalto', True) | first }}"
```

```suggestion
          | selectattr('default') | first }}"
```
https://jinja.palletsprojects.com/en/2.9.x/templates/#selectattr
"If no test is specified, the attribute’s value will be evaluated as a boolean."